### DISCUSSION TITLE
Conductor ui: connection to failed

### DISCUSSION THREAD

**Discussion by pankajthamkeappdirect** [Netflix/conductor#2945] [Q&A] [answered]
Conductor ui: connection to failed due to elastic search

**Comment by peterlau** [comment]
That's an error associated with the React `react-scripts` hotloading functionality. It should be unrelated to the issue you're encountering. Could you switch to the `Network` tab in the Chrome debugger with the Fetch/XHR filter activated, and post a screenshot of the calls?

**Comment by peterlau** [comment, marked as answer]
The reason I asked to see that is to ascertain whether it is a UI based problem. In this case the search API is correctly returning - the fact that it is an empty response points to elasticsearch indexing.
